### PR TITLE
Move config files to resources in Kinesis

### PR DESCRIPTION
**Pull request by ebyhr**
## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

**Review by wendigo — APPROVED** [review]

**Comment by wendigo** [comment]
Seems like they are used:
```
 TestKinesisTableDescriptionSupplier.testRelatedObjects:89 » Trino Table description location does not exist or is not a directory
Error:    TestKinesisTableDescriptionSupplier.testTableDefinition:70 » Trino Table description location does not exist or is not a directory
```

**Review by wendigo — CHANGES_REQUESTED** [review]
Used.

File: plugin/trino-kinesis/src/test/resources/etc/catalog/kinesis.properties [hunk 1]
```diff
@@ -31,4 +31,3 @@ kinesis.checkpoint-enabled=false
 #kinesis.checkpoint-interval-ms=60000ms
 #kinesis.checkpoint-logical-name=process1
 #kinesis.iterator-number=0
-
```
Comment (ebyhr) [inline plugin/trino-kinesis/src/test/resources/etc/catalog/kinesis.properties:34]: Redundant empty lines cause check style failure.

**Comment by ebyhr** [comment]
@wendigo Moved the files to resources directory instead. Please take another look.

**Review by wendigo — APPROVED** [review]

**Pull request merged**